packages/backend/src/services/UnfurlService/UnfurlService.ts

```diff
@@ -217,6 +217,7 @@ export class UnfurlService extends BaseService {
             chartType,
             resourceUuid,
             chartTileUuids: rest.chartTileUuids,
+            sqlChartTileUuids: rest.sqlChartTileUuids,
```

Comment:
was missing this value being passed to the main screenshot. Was only waiting properly for sql charts when there was a retryable error, like protocol error, or timeout error